### PR TITLE
send python coverage to codecov, remove buildpulse which was not helping

### PR DESCRIPTION
**Pull request by anshbansal**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Comment by anshbansal** [comment]
Got this so far
image

before last commit's coverage was uploaded

File: .github/workflows/metadata-ingestion.yml [hunk 1]
```diff
@@ -69,17 +69,16 @@ jobs:
...
         with:
-          account: ${{ secrets.BUILDPULSE_ACCOUNT }}
-          repository: ${{ secrets.BUILDPULSE_REPO }}
-          path: |
-            **/junit*.xml
-          key: ${{ secrets.BUILDPULSE_ACCESS_KEY_ID }}
-          secret: ${{ secrets.BUILDPULSE_SECRET_ACCESS_KEY }}
+          token: ${{ secrets.CODECOV_TOKEN }}
+          directory: .
+          fail_ci_if_error: true
```
Comment (shirshanka) [inline .github/workflows/metadata-ingestion.yml:78]: what does this flag do?

**Review by shirshanka — COMMENTED** [review]
Overall seems fine.. have a question about the ci flag

File: .github/workflows/metadata-ingestion.yml (hunk 1 above)
Comment (anshbansal) [reply .github/workflows/metadata-ingestion.yml:78]: In case the code coverage fails to be sent this will fail this workflow

**Review by hsheth2 — APPROVED** [review]

**Pull request merged**